clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/EventHandler.java

```diff
@@ -57,29 +57,12 @@ public interface EventHandler extends Closeable {
      *
      * See {@link CompletableApplicationEvent#get(Timer)} and {@link Future#get(long, TimeUnit)} for more details.
      *
-     * @param event     An {@link CompletableApplicationEvent} created by the polling thread.
-     * @param timeoutMs Value in milliseconds for which to wait for the event to complete
-     * @return          Value that is the result of the event
-     * @param <T>       Type of return value of the event
+     * @param event An {@link CompletableApplicationEvent} created by the polling thread.
```

Comment:
typo: "A CompletableF..." instead of "An CompletableF..."

Reply:
Fixed typo.